recipes/pytest-executable/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pytest_executable" %}
+{% set version = "0.4.0" %}
+
+package:
+  name: pytest-executable
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 316b7982bcdb923ad6b8453f54b348bf81328dcdcecad212034acdf592bd3a4d
+
+build:
+  number: 0
+  include_recipe: False
+  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python >=3.6
+    - pytest >=4.3,<6
+    - pyyaml >=3,<6
+    - jsonschema >=2,<4
+    - jinja2 >=2.7,<3
+    - dataclasses  # [py<37]
```

Comment:
I don't think you can use `noarch: python` with a selector

Reply:
But how could I do this?

I already asked [here](https://github.com/conda-forge/staged-recipes/pull/11636#issuecomment-629463006).

Reply:
It would appear dataclasses is in fact a no-op on pythons other than 3.6:

https://github.com/conda-forge/dataclasses-feedstock/blob/master/recipe/meta.yaml#L15

Since conda-forge no longer builds `py27`, it would appear it is safe to use `dataclasses` without selector in `noarch: python`. To be certain, you'd want to pin `python >=3.6`, which you have done, so no problem here!

Reply:
Thanks, I removed the selector then.
BTW, funny to see the dataclasses recipe does have selectors!

Reply:
But the dataclasses recipe does not have noarch, at last I got it!